Server/VikingVault.DataAccess/Migrations/VikingVaultDbContextModelSnapshot.cs

```diff
@@ -137,14 +137,18 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<DateTime>("Date");
 
-                    b.Property<string>("OtherParty");
+                    b.Property<string>("Details");
+
+                    b.Property<int?>("SenderOrReceiverId");
```

Comment:
This basically means that a transaction can exist without sender/receiver? Doesn`t sound good to me.

Reply:
If I make it required, this happens: Introducing FOREIGN KEY constraint 'FK_Transactions_User_SenderOrReceiverId' on table 'Transactions' may cause cycles or multiple cascade paths. Specify ON DELETE NO ACTION or ON UPDATE NO ACTION, or modify other FOREIGN KEY constraints.

Reply:
You don`t need to have on cascade delete on all your foreign keys. Think about how can you write everything so that it doesn`t give you that error anymore.

Reply:
Already tried. Can't make it work. Would appreciate some help

Reply:
Done